public/carbonads.css

```diff
@@ -31,7 +31,7 @@
 }
 
 #carbonads a {
-  color: inherit;
+  color: var(--text-color-light);
   text-decoration: none;
 }
 #carbonads a:hover {
```

Comment:
Hey 👋 

This one needs to be changed as well, otherwise on hover the font will be invisible.